sigopt/xgboost/run.py

```diff
@@ -133,9 +134,8 @@ def log_metadata(self):
   def log_params(self):
     for name in self.params.keys():
```

Comment:
`for` is not needed.

Reply:
I think you want `self.run.params.setdeefaults({name: self.params[name]})`